tests/unit/test_windows_compatibility.py

```
@@ -185,7 +189,7 @@ async def test_windows_script_execution_no_chmod(
     @patch("tempfile.NamedTemporaryFile")
     @patch("os.chmod")
     async def test_unix_script_execution_with_chmod(
-        self, mock_chmod, mock_tempfile, mock_platform
+        self, mock_chmod, mock_tempfile, _mock_platform
```

Pylint (3.3.7)

[warning] Unused argument 'mock_chmod'

(W0613)